client/src/pages/index.tsx

```diff
@@ -29,7 +42,15 @@ export default function Home() {
   if (showSplash) {
     return <SplashScreen />;
   }
+  return (
+    <main
+      className={cn("flex min-h-screen flex-col font-sans", fontSans.variable)}
```

Comment:
Font is defined elsewhere

Reply:
`Styles.css` have been merged so `fontSans` won't be necessary I think. Can remove all reference to fonts in the `className` attributes.